NomaiGrandPrix/SpawnActionFactory.cs

```diff
@@ -9,7 +9,8 @@ public class SpawnActionFactory
         private static Dictionary<string, Action[]> actionsMap = new Dictionary<string, Action[]>
         {
             { "Spawn_Module_Sunken", new Action[] { OpenSunkenModuleAirlock } },
-            { "Spawn_Module_Intact", new Action[] { OpenProbeCannonAirlocks } }
+            { "Spawn_Module_Intact", new Action[] { OpenProbeCannonAirlocks } },
+            { "Spawn_TH_ZeroGCave", new Action[] { LowerZeroGElevator } }
```

Comment:
```suggestion
            { "Spawn_TH_ZeroGCave", new Action[] { LowerZeroGElevator } },
```
A trailing comma here can keep future diffs cleaner.